### PR TITLE
SHOT-3781: Scene Data Validation

### PR DESCRIPTION
**Pull request by staceyoue**
- Create class AliasSceneDataValidator
- Add hooks for new tk-multi-data-validation app
- Add python/api module for Alias Python API utility methods

**Requires Alias Python API #98**

### CODE REVIEW

File: startup.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ class AliasLauncher(SoftwareLauncher):
     CODE_NAMES = {
         "AutoStudio": dict(flags="-a as", icon="icon_as_256.png"),
         "Surface": dict(flags="-a ss", icon="icon_ss_256.png"),
-        "Design": dict(flags="-a ds", icon="icon_cs_256.png"),
```
Comment (rob-aitchison) [inline startup.py:31]: I believe this icon has been removed as the product no longer exists.

**Review by rob-aitchison — CHANGES_REQUESTED** [review]
Please check on that icon. Everything else looks great with a quick skim; will read thoroughly once we release it - thanks!

File: startup.py (hunk 1 above)
Comment (staceyoue) [reply startup.py:31]: I didn't actually make that change -- it was from a merge with master. I will just remove that change from this branch

**Review by rob-aitchison — APPROVED** [review]
👍🏻

**Pull request merged**